lib/node_modules/@stdlib/array/log/docs/types/index.d.ts

```diff
@@ -0,0 +1,43 @@
+/*
+* @license Apache-2.0
+*
+* Copyright (c) 2022 The Stdlib Authors.
+*
+* Licensed under the Apache License, Version 2.0 (the "License");
+* you may not use this file except in compliance with the License.
+* You may obtain a copy of the License at
+*
+*    http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*/
+
+// TypeScript Version: 2.0
+
+/**
+* Insert array element values into a format string and print the result.
+*
+* @param str - format string
+* @param args - collections or values
+* @throws {TypeError} first argument must be a string
```

Comment:
This line is not needed, as passing a non-string value is not allowed in TypeScript. Meaning, there will be a compiler error. We only document errors that a compiler would not catch, such as L27.